src/settings/BatchGroupConfigurationSettings/BatchGroupCongiurationSettingsForm.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import stripesFinalForm from '@folio/stripes/final-form';
+import {
+  Button,
+  KeyValue,
+  Pane,
+  PaneFooter,
+  Select,
+} from '@folio/stripes/components';
+
+const BatchGroupConfigurationSettingsForm = ({
+  batchGroups,
+  batchGroupsOptions,
+  handleSubmit,
+  paneTitle,
+  pristine,
+  setBatchGroupId,
+  submitting,
+}) => {
+  const getPaneFooter = () => {
+    const start = (
+      <Button
+        buttonStyle="default mega"
+        data-test-run-manual-export-button
+        disabled
+      >
+        <FormattedMessage id="ui-invoice.button.runManualExport" />
+      </Button>
+    );
+
+    const end = (
+      <Button
+        buttonStyle="primary mega"
+        data-test-save-button
+        disabled={pristine || submitting}
+        onClick={handleSubmit}
+        type="submit"
+      >
+        <FormattedMessage id="ui-invoice.button.save" />
+      </Button>
+    );
+
+    return (
+      <PaneFooter
+        renderStart={start}
+        renderEnd={end}
+      />
+    );
+  };
+
+  return (
+    <Pane
+      data-test-batch-group-configuration-settings
+      defaultWidth="fill"
+      footer={getPaneFooter()}
```

Comment:
Personally, I believe calling functions at render is not the best way. Why it's a function at the first place?

Reply:
Ok, created component for it